packages/cbioportal-utils/src/ptm/PtmUtils.ts

```diff
@@ -0,0 +1,368 @@
+import * as _ from 'lodash';
+
+import { PostTranslationalModification as DbPtm } from 'genome-nexus-ts-api-client';
```

Comment:
We should probably rename `PostTranslationalModification` to `DbPtm` on the GenomeNexus side